Client profile page

aaleksaaleksic opened the pull request:
Closes : #97 

This is what I imagined what a client profile page should look like. I made it like a component in case we will use it for the same purpose but for employee if he wants to look at his details, maybe I'm wrong but it's a quick fix either way. Let me know what you think!!! 💯

markocic reviewed (changes requested):
This is a good start. We can put these fields in two columns seeing as there is a lot of empty space. Try that and let's see how it looks.

aaleksaaleksic commented:
Will notify you when I do that!

aaleksaaleksic commented:
I made everything in two collums instead of one to save space. 

I added a loading spinner while data is loading. 

Aaaaand I played a little bit with effects :)))) 

I added a blur/shadow gray effect to spice things up, I know we are trying to follow some kind of style but it looked awesome to me `especially in dark mode` , please test it and see it for yourself.


I will understand if I have to remove the effect, it will be an easy job, but I just wanted your opinion on it because it looked cool to me! 

Let me know what you think 💯

markocic reviewed (changes requested):
I think it looks awesome! We can leave this like this. Although I think it looks a bit better if you remove the black "border" around the page. Feel free to remove that padding that was set in the `app/c/layout.tsx:28`, specifically class `p-4`.

aaleksaaleksic commented:
Sure, why not. Will do! I'll notify you when I do! :)

aaleksaaleksic commented:
I did the corrections! Now it should be ready for merge!
